### PR TITLE
Release notes v1.0.0

### PR DESCRIPTION
**Pull request by brimoor**
## Summary by CodeRabbit

- **New Features**
	- Introduced FiftyOne Teams 2.1.0 with super admin capabilities for deployment migration and user invitations.
	- Released FiftyOne 1.0.0 featuring open-sourced FiftyOne Brain, Modal Panels, and enhanced dataset management.
	- Added support for loading Med-SAM2 in the model zoo.
	- Improved UI and documentation with numerous built-in operators and enhancements.

- **Bug Fixes**
	- Fixed BSON serialization bug affecting dataset view cloning and exporting.

- **Documentation**
	- Updated release notes to reflect new features and improvements.

### CODE REVIEW

**Review by findtopher — APPROVED** [review]
sure?

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -3,14 +3,115 @@ FiftyOne Release Notes
...
+
+FiftyOne 1.0.0
+--------------
+*Released October 1, 2024*
+
+What's New
+
+- The `FiftyOne Brain <https://github.com/voxel51/fiftyone-brain>`_ is now
+  fully open source. Contributions are welcome!
+- Introducing Modal Panels, bringing the ability to develop and use panels in
```
Comment (brimoor) [inline docs/source/release-notes.rst:31]: TODO: add a docs link here

File: docs/source/release-notes.rst [hunk 2]
```diff
@@ -3,14 +3,115 @@ FiftyOne Release Notes
...
+
+- Added dozens of :ref:`builtin operators <using-operators>` for performing
+  common operations directly from the App
+  `#4830 <https://github.com/voxel51/fiftyone/pull/4830>`_
+- Label overlays in the grid are now scaled proportionally to grid zoom
+  `#4747 <https://github.com/voxel51/fiftyone/pull/4747>`_
+- Improved support for visualizing and filtering |DynamicEmbeddedDocument| list
+  fields
+  `#4833 <https://github.com/voxel51/fiftyone/pull/4833>`_
+- Added a new timeline API for synchronizing playback of multiple modal panels
```
Comment (brimoor) [inline docs/source/release-notes.rst:60]: TODO: add a docs link here?

File: docs/source/release-notes.rst (hunk 1 above)
Comment (ritch) [reply docs/source/release-notes.rst:31]: https://github.com/voxel51/fiftyone/pull/4847#pullrequestreview-2329648593

**Review by benjaminpkane — APPROVED** [review]
LGTM

**Review by findtopher — APPROVED** [review]
sure thing!

**Pull request merged**